Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/apiclient'...
Listing './lib/python3.5/site-packages/dateutil'...
Listing './lib/python3.5/site-packages/dateutil/tz'...
Listing './lib/python3.5/site-packages/dateutil/zoneinfo'...
Listing './lib/python3.5/site-packages/docopt-0.6.2.egg-info'...
Listing './lib/python3.5/site-packages/google_api_python_client-1.6.4.dist-info'...
Listing './lib/python3.5/site-packages/googleapiclient'...
Listing './lib/python3.5/site-packages/googleapiclient/discovery_cache'...
Listing './lib/python3.5/site-packages/gsync-0.1.14.egg-info'...
Listing './lib/python3.5/site-packages/httplib2'...
Listing './lib/python3.5/site-packages/httplib2-0.10.3.egg-info'...
Listing './lib/python3.5/site-packages/libgsync'...
Compiling './lib/python3.5/site-packages/libgsync/crawler.py'...
***   File "./lib/python3.5/site-packages/libgsync/crawler.py", line 152
    except KeyboardInterrupt, e:
                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/drive'...
Compiling './lib/python3.5/site-packages/libgsync/drive/__init__.py'...
***   File "./lib/python3.5/site-packages/libgsync/drive/__init__.py", line 287
    os.mkdir(configdir, 0700)
                           ^
SyntaxError: invalid token

Compiling './lib/python3.5/site-packages/libgsync/drive/mimetypes.py'...
***   File "./lib/python3.5/site-packages/libgsync/drive/mimetypes.py", line 17
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/options'...
Compiling './lib/python3.5/site-packages/libgsync/output.py'...
***   File "./lib/python3.5/site-packages/libgsync/output.py", line 82
    self.bytesWritten = 0L
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/sync'...
Compiling './lib/python3.5/site-packages/libgsync/sync/__init__.py'...
***   File "./lib/python3.5/site-packages/libgsync/sync/__init__.py", line 15
    totalBytesSent = 0L
                      ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/sync/file'...
Compiling './lib/python3.5/site-packages/libgsync/sync/file/__init__.py'...
***   File "./lib/python3.5/site-packages/libgsync/sync/file/__init__.py", line 50
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/sync/file/local'...
Compiling './lib/python3.5/site-packages/libgsync/sync/file/local/__init__.py'...
***   File "./lib/python3.5/site-packages/libgsync/sync/file/local/__init__.py", line 60
    except OSError, e:
                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/libgsync/sync/file/remote'...
Listing './lib/python3.5/site-packages/oauth2client'...
Listing './lib/python3.5/site-packages/oauth2client/contrib'...
Listing './lib/python3.5/site-packages/oauth2client/contrib/django_util'...
Listing './lib/python3.5/site-packages/oauth2client-4.1.2.dist-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pyasn1'...
Listing './lib/python3.5/site-packages/pyasn1/codec'...
Listing './lib/python3.5/site-packages/pyasn1/codec/ber'...
Listing './lib/python3.5/site-packages/pyasn1/codec/cer'...
Listing './lib/python3.5/site-packages/pyasn1/codec/der'...
Listing './lib/python3.5/site-packages/pyasn1/codec/native'...
Listing './lib/python3.5/site-packages/pyasn1/compat'...
Listing './lib/python3.5/site-packages/pyasn1/type'...
Listing './lib/python3.5/site-packages/pyasn1-0.4.2.dist-info'...
Listing './lib/python3.5/site-packages/pyasn1_modules'...
Listing './lib/python3.5/site-packages/pyasn1_modules-0.2.1.dist-info'...
Listing './lib/python3.5/site-packages/python_dateutil-2.6.1.dist-info'...
Listing './lib/python3.5/site-packages/rsa'...
Listing './lib/python3.5/site-packages/rsa-3.4.2.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/six-1.11.0.dist-info'...
Listing './lib/python3.5/site-packages/uritemplate'...
Listing './lib/python3.5/site-packages/uritemplate-3.0.0.dist-info'...
Listing './lib/python3.5/site-packages/urllib3'...
Listing './lib/python3.5/site-packages/urllib3/contrib'...
Listing './lib/python3.5/site-packages/urllib3/contrib/_securetransport'...
Listing './lib/python3.5/site-packages/urllib3/packages'...
Listing './lib/python3.5/site-packages/urllib3/packages/backports'...
Listing './lib/python3.5/site-packages/urllib3/packages/ssl_match_hostname'...
Listing './lib/python3.5/site-packages/urllib3/util'...
Listing './lib/python3.5/site-packages/urllib3-1.22.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
